vmdb/lib/vmdb/permission_stores/yaml.rb

```diff
@@ -0,0 +1,23 @@
+require 'psych'
+
+module Vmdb
+  module PermissionStores
+    def self.create(config)
+      YAML.new config.options[:filename]
+    end
+
+    class YAML
+      def initialize(file)
+        @permissions = Psych.load_file file
+      end
+
+      def can?(permission)
```

Comment:
I don't know why, but I find this method name confusing...it's just reads funny to me.  `store.can?(permission)`